server/utils.go

```diff
@@ -166,3 +166,24 @@ func parseJIRAIssuesFromText(text string, keys []string) []string {
 
 	return issues
 }
+
+func formatMarkdownLink(text string, link string) string {
+	return fmt.Sprintf("[%s](%s)", text, link)
+}
+
+func findIssuesInLinks(text string, instanceUrl string) []string {
```

Comment:
Could you please write unit tests for this function so we can see its behavior under different circumstances?

Reply:
I actually wrote unit tests for this but I don't know why at some point the "utils_test" file got removed. I can upload it again if there's no problem with that

Reply:
> at some point the "utils_test" file got removed

Some file structure has changed. The `utils_test.go` file is now located at `server/utils/utils_test.go`